app/styles/app.scss

```diff
@@ -1,9 +1,34 @@
-@import 'ember-paper';
+.modal {
+  display: flex;
+  visibility: hidden;
+  opacity: 0;
+  transition: visibility 0s linear 1s, opacity 1s;
+}
+
+.is-active {
+  visibility: visible;
+  opacity: 1;
+  transition: visibility 0s linear 0s, opacity 1s;
+}
+
+.main-navbar {
+  margin: 0 5%;
+}
+
+.organization-card:hover {
+  box-shadow: 5px 5px 3px 2px rgba(10,10,10,.1), 0 0 0 1px rgba(10,10,10,.1);
+  cursor: pointer;
+}
+
+.org-icon {
+  max-height: 25px;
+  max-width: 25px;
+}
 
-.gsoc-organizations-list {
-  height: 100vh;
+.card-content {
+  overflow-wrap: break-word;
 }
 
-.search-results {
-  overflow-y: scroll;
+.tag {
+  text-shadow: 1px 1px 1px white;
```

Comment:
The text shadow makes it harder to read.
![image](https://user-images.githubusercontent.com/1438859/41984792-bf0dbbe4-7a5b-11e8-88fc-5e752d860ad5.png)

Reply:
Oh but it's also even harder when there's no text-shadow.

Maybe just use label color as `border-color` for now? If it's too hard to implement dynamic text color.

Reply:
Browser support is vary, the proper implementation is still in a draft proposal, the support is small. The other solution involves some javascript.
Maybe, i'm missing something, can you specify the link?

Reply:
Yes, use Javascript.